practice-two/src/index.html

```diff
@@ -298,5 +298,65 @@ <h2 class="enterprise-title">ENTERPRISE</h2>
       </div>
     </section>
     <!-- END BOOST-BUSINESS -->
+
+    <!-- START FOOTER -->
+    <footer class="site-footer">
+      <div class="general-container">
+        <div class="footer-group">
+          <div class="footer-wrap">
+            <a href="javascript:void(0);" class="footer-branch-logo">
+              <div class="footer-container">
+                <img src="./assets/logo.png" class="footer-logo" alt="footer-logo" />
+              </div>
+              <h1 class="footer-branch-name">Product</h1>
+            </a>
+            <a href="javascript:void(0)" class="footer-social-network">
+              <div class="social-network">
+                <div class="footer-container">
+                  <img src="./assets/instagram.png" class="logo-social-network" alt="instagram" />
+                </div>
+                <div class="footer-container">
```

Comment:
Footer container mean it wrap all footer's content, not a part of footer. Pls try another name